types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -8,7 +13,7 @@ interface UseChatUtilsHook {
   goToMoveChat: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `goToContactScreen` function signature is also correct. It now accepts an object with a `from` property, allowing the navigation to be aware of whether the user is anonymous or signed-in.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->